x11-misc/polybar/polybar-3.2.1.ebuild

```diff
@@ -62,6 +63,7 @@ src_configure() {
 		-DENABLE_MPD="$(usex mpd)"
 		-DENABLE_NETWORK="$(usex network)"
 		-DENABLE_PULSEAUDIO="$(usex pulseaudio)"
+		-DCMAKE_INSTALL_DOCDIR="share/doc/${PF}"
```

Comment:
Why does this needed to passed through? Can this be achieved through GNUInstallDirs cmake module?

Reply:
I don't think GNUInstallDirs can be used (but I don't know too much about it). See https://cmake.org/cmake/help/v3.4/module/GNUInstallDirs.html where `DOCDIR` points to `DATAROOTDIR/doc/PROJECT_NAME` which is not versioned.

It needs to be passed for the same reason. The previous `DOCDIR` is not versioned.